api/src/main/java/pl/kolendateam/dadcard/characterCard/dto/CharacterDTO.java

```diff
@@ -14,20 +18,26 @@
 public class CharacterDTO {
     public String characterName;
     public String playerName;
+    public AbilitysDTO abilitys;
     public ArrayList<ClassPcListDTO> classPcList;
     public int ecl;
-    public int bab;
+    public VitalityDTO vitality;
+    public int streghtAttack;
+    public int dextrityAttack;
     public SavingThrowDTO savingThrows;
     public double skillPoints;
     public ArrayList <SkillsDTO> skillsList;
 
     public CharacterDTO(Character character) {
         this.characterName = character.getCharacterName();
         this.playerName = character.getPlayerName();
+        this.abilitys = MapperAbilitysToDTO.toAbilityDTO(character.getAbilitys());
         this.classPcList = MapperClassPcListToDTO.toClassPcListDTO(character.getClassPcArray());
         this.ecl = character.getEcl();
-        this.bab = (int) character.getBab();
-        this.savingThrows = MapperSavingThrowToDTO.toSavingThrowDTO(character.getSavingThrow());
+        this.vitality = MapperVitalityToDTO.toVitalityDTO(character.getVitality());
+        this.streghtAttack = (int) character.getBab()+character.getAbilitys().getStreghtBonus();
```

Comment:
unikamy łancuchów get. tzn. ze jesli masz root np. character to nie powinienes wyciagac z niego skladowej z niej skladaowej i w tedy prosic o jakies obliczenia. Tylko character powinien miec na to metody odrazu

Reply:
poprawie